include/boost/geometry/strategy/geographic/area.hpp

```diff
@@ -80,6 +80,9 @@ class geographic
 protected :
     struct spheroid_constants
     {
+        // Area formula is implemented for a maximum series order 5
+        static constexpr auto SeriesOrderNorm = SeriesOrder > 5 ? 5 : SeriesOrder;
```

Comment:
Why not define it in the class scope together with other static const variables?

Reply:
thanks I overlooked it!